core/src/androidMain/kotlin/com/revenuecat/purchases/kmp/di/AndroidProvider.kt

```diff
@@ -56,7 +56,25 @@ internal object AndroidProvider : Application.ActivityLifecycleCallbacks {
 
 internal fun AndroidProvider.requireApplication(): Application =
     application ?: error(
-        "Purchases has no reference to the Application. This is a bug in the library."
+        "Purchases has no reference to the Application. Please make sure the merged " +
+                "AndroidManifest.xml contains the InitializationProvider element like below, " +
+                "with at least the PurchasesInitializer child. This should normally happen " +
+                "automatically." +
+                "\n\n" +
+                """
+        <provider
+            android:name="androidx.startup.InitializationProvider"
```

Comment:
I'm wondering, if they need to add this to their manifest, whether they would also need to add the androix library to their gradle files...

Reply:
Hmm that shouldn't be necessary. The library will be transitively provided by purchases-kmp.